test/remote/gateways/remote_braintree_blue_test.rb

```diff
@@ -2,7 +2,8 @@
 
 class RemoteBraintreeBlueTest < Test::Unit::TestCase
   def setup
-    @gateway = BraintreeGateway.new(fixtures(:braintree_blue))
+    fixture_key = method_name.match?(/bank_account/i) ? :braintree_blue_with_ach_enabled : :braintree_blue
```

Comment:
could you please explain me a little better this statement?

Reply:
Ahhh @jherreraa  is very simple => the 'setup' method runs before every single test is going to be run, so it checks the name of the test, if the name of the test has the 'bank_account' string on it, it is going to set Braintree credentials with the account that has ACH support enabled.